stdlib/3/tempfile.pyi

```diff
@@ -74,6 +74,9 @@ if sys.version_info >= (3, 5):
 
     def mkstemp(suffix: Optional[AnyStr] = ..., prefix: Optional[AnyStr] = ..., dir: Optional[AnyStr] = ...,
                 text: bool = ...) -> Tuple[int, AnyStr]: ...
+    @overload
+    def mkdtemp() -> str: ...
```

Comment:
On the train right now, so I can't check, but I suspect that we should add `dir` as a keyword-only argument to this overload.

Reply:
Based on my reading of the docs, `dir` should be `AnyStr`. Fixed!